plugin/trino-openlineage/pom.xml

```diff
@@ -123,6 +123,12 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <scope>runtime</scope>
```

Comment:
> fix open lineage runtime failure with HTTP transport

Please change the commit title to "Add commons-lang3 to OpenLineage plugin" and mention the reason in the commit body.